pkg/controller/remotemachineset/remotemachineset_controller.go

```diff
@@ -87,6 +87,13 @@ func Add(mgr manager.Manager) error {
 		return err
 	}
 
+	// Watch for changes to MachinePoolNameLeases
+	err = c.Watch(&source.Kind{Type: &hivev1.ClusterDeployment{}}, &handler.EnqueueRequestsFromMapFunc{
```

Comment:
Remove this. This is an exact replica of the previous block of code, except for an incorrect change to the comment.